app/src/main/java/edu/wpi/first/shuffleboard/app/Main.java

```diff
@@ -16,7 +20,10 @@
   private static final Logger logger = Logger.getLogger(Main.class.getName());
 
   @SuppressWarnings("JavadocMethod")
-  public static void main(String[] args) {
+  public static void main(String[] args) throws IOException {
+    WPIUtilJNI.Helper.setExtractOnStaticLoad(false);
+    CombinedRuntimeLoader.loadLibraries(Main.class, "wpiutiljni");
```

Comment:
Why does this have to be called in Main?

Reply:
It's the base libs, so it either needs to be called in main or at the very beginning when javafx initialized. Main seemed like the better option since it's the actual entry point.

Reply:
Part of this change is that automatic extraction will no longer happen anymore. This portion loads wpiutil, which then allows the plugins to load their own libraries